markdownhints/build.gradle

```diff
@@ -16,6 +16,10 @@ android {
       proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
   }
+
+  lintOptions {
+    abortOnError false
```

Comment:
:'(

Reply:
I’d be glad to revert that change but was unable to get rid of the errors that are currently being reported. If you look at the previous logs for this branch you’ll see that there are issues with flexmark. Perhaps you might have an idea how to fix that. If so we can refer that.

Reply:
I'll try to put together a PR to resolve them shortly.

Reply:
I filed #151 to silence the flexmark warning since there's nothing on the internet that'll resolve this properly. We have the option of forcibly unloading the problematic classes with R8 but I'm not sure if that's a good idea. Another choice can probably be just dropping flexmark and using https://github.com/atlassian/commonmark-java.